.github/workflows/test.yml

```
@@ -50,23 +50,14 @@ jobs:
       - name: "Install Dependencies"
         run: |
           sudo apt install wait-for-it -y
-
-      - name: "Deploy conda-store-server docker container"
-        run: |
-          docker-compose -f docker-compose-dev.yml up -d --build
-          docker ps
-
-          wait-for-it localhost:8080 # conda-store-server
+          playwright install chromium
+          yarn install --immutable
+          yarn run build
 
       - name: "Deploy webpack dev server"
```

```suggestion
      - name: "Start web server and run tests"
```

Let's use a more descriptive name